projects/bp-gallery/src/components/views/location-curating/FoldoutStatusContext.ts

```diff
@@ -0,0 +1,14 @@
+import { MutableRefObject, createContext, useContext } from 'react';
+
+export const FoldoutStatusContext = createContext<MutableRefObject<
+  | {
+      [key: string]: {
+        isOpen: boolean;
+      };
+    }
```

Comment:
Maybe extract this into its own (exported) type so it can be used in `FoldoutStatusProvider.tsx`